airbyte-integrations/connectors/source-square/acceptance-test-config.yml

```diff
@@ -42,6 +42,11 @@ acceptance_tests:
             bypass_reason: "Floating data"
           - name: updated_at
             bypass_reason: "Floating data"
+        categories:
```

Comment:
How is this related to the oauth change?

Reply:
This was needed to fix buid, not actually related to oauth changes